tools/git/README.md

```
@@ -0,0 +1,76 @@
+<div dir="rtl">
+
+# ‏tools/git — שני שומרים לתהליך
+
+הסקריפטים כאן אינם נוחות. כל אחד מהם נולד מכשל שקרה, ושניהם מממשים את
+אותו עיקרון: **פעולה שלא הצליחה לבדוק — נכשלת, לא מוותרת** (עיקרון 5).
+
+---
+
+## ‏`lab-work-checkout.sh` — עץ עבודה על המעבדה שאינו יכול לרשת ref מיושן
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)